OpenRA.Game/Server/Server.cs

```diff
@@ -169,14 +108,14 @@ public Server(IPEndPoint endpoint, ServerSettings settings, ModData modData)
 
 					foreach (var s in checkRead)
 						if (s == listener.Server) AcceptConnection();
-						else if (PreConns.Count > 0)
+						else if (preConns.Count != 0)
```

Comment:
It **can** be less than 0 (if the check allows it) but it should never be, in practice.